### PR TITLE
add a full script smoke test

### PR DESCRIPTION
**Pull request by aslonnie**
with some basic test data.

### CODE REVIEW

File: ray_ci/test_pipeline_ci.py [hunk 1]
```diff
@@ -230,5 +233,49 @@ def test_read_lineline():
...
+        "RUNNER_QUEUE_GPU_NORM": "gpu",
+        "RUNNER_QUEUE_GPU_LARGE": "gpularge",
+        "BUCKET_PATH": "s3://bucket",
+        "RAY_REPO_DIR": "/ray",
+        "PIPELINE_REPO_DIR": "/pipeline",
+
+        "RAY_CI_PYTHON_DEPENDENCIES_AFFECTED": "1",
+        "BUILDKITE_PULL_REQUEST_REPO": "ray",
+        "BUILDKITE_BRANCH": "master",
+        "BUILDKITE_COMMIT": "abcd1234",
```
Comment (can-anyscale) [inline ray_ci/test_pipeline_ci.py:255]: Is this hard-coded?

File: ray_ci/test_pipeline_ci.py [hunk 2]
```diff
@@ -230,5 +233,49 @@ def test_read_lineline():
...
+        "REMOTE_CACHE_URL": "https://remote-cache",
+        "ECR_REPO": "ecr_repo",
+        "ECR_BASE_REPO": "ecr_base_repo",
+        "RUNNER_QUEUE_DEFAULT": "default",
+        "RUNNER_QUEUE_SMALL": "small",
+        "RUNNER_QUEUE_MEDIUM": "medium",
+        "RUNNER_QUEUE_LARGE": "large",
+        "RUNNER_QUEUE_GPU_NORM": "gpu",
+        "RUNNER_QUEUE_GPU_LARGE": "gpularge",
+        "BUCKET_PATH": "s3://bucket",
```
Comment (can-anyscale) [inline ray_ci/test_pipeline_ci.py:248]: valid path?

**Review by can-anyscale — COMMENTED** [review]
Nice, nice. How do we test this?

File: ray_ci/test_pipeline_ci.py (hunk 1 above)
Comment (aslonnie) [reply ray_ci/test_pipeline_ci.py:255]: this is to provide the required env var input for the script to run.

File: ray_ci/test_pipeline_ci.py (hunk 2 above)
Comment (aslonnie) [reply ray_ci/test_pipeline_ci.py:248]: not sure what the question mean. it is not a real bucket.

**Comment by aslonnie** [comment]
> How do we test this?

it is in the pipeline: https://buildkite.com/ray-project/rayci/builds/22#01890f08-fd8e-4b60-8363-38a248a6a504/156-279

**Review by can-anyscale — APPROVED** [review]
Ohh, it's test data. I thought it's actual logic. What was I thinking?

**Pull request merged**